panic in load embedded ruleguard rules: can't load fmt

I was trying out the latest v0.6.0 (https://github.com/go-critic/go-critic/releases/tag/v0.6.0) on Windows and it failed with:

```
> gocritic ./...
panic: load embedded ruleguard rules: rules/rules.go:13: can't load fmt

goroutine 1 [running]:
github.com/go-critic/go-critic/checkers.init.9()
        /home/isharipov/CODE/go-critic3/checkers/checkers.go:47 +0x3b4
```

--- a/go.mod
+++ b/go.mod
@@ -16,7 +16,7 @@ require (
 	github.com/mattn/goveralls v0.0.2
 	github.com/pborman/uuid v1.2.0 // indirect
 	github.com/quasilyte/go-consistent v0.0.0-20190521200055-c6f3937de18c
-	github.com/quasilyte/go-ruleguard v0.3.12
+	github.com/quasilyte/go-ruleguard v0.3.13
 	github.com/quasilyte/go-ruleguard/dsl v0.3.10
 	github.com/quasilyte/regex/syntax v0.0.0-20200407221936-30656e2c4a95
 	golang.org/x/tools v0.0.0-20201230224404-63754364767c

